buildkite/src/Command/DeployTestnet.dhall

```diff
@@ -5,9 +5,10 @@ let Size = ./Size.dhall
 
 let Cmd = ../Lib/Cmds.dhall
 
-let defaultArtifactStep = { name = "MinaArtifact", key = "mina-docker-image" }
 
-let deployEnv = "DOCKER_DEPLOY_ENV" in
+let deployEnv = "DOCKER_DEPLOY_ENV"
+-- testnet artifacts include: genesis ledgers, block producer keys,...
+let testnetArtifactPath = "/tmp/artifacts" in
```

Comment:
Not sure if worth making an input var...though thinking of just creating a TODO/issue for creating sane defaults for all input vars (where makes sense)